crypto/rand/drbg_lib.c

```diff
@@ -164,7 +224,18 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
     }
 
     drbg->state = DRBG_READY;
-    drbg->reseed_counter = 1;
+    drbg->generate_counter = 0;
+
+    if (drbg->reseed_time != 0) {
+        /* automatic reseeding is enabled */
+        if (drbg->parent == NULL) {
+            drbg->reseed_time = time(NULL);
+            drbg->reseed_counter = 0;
+        } else {
+            drbg->reseed_time = drbg->parent->reseed_time;
```

Comment:
Does the child still need to track the parent's reseed_time now that we have a reseed_counter?

Reply:
So I guess the answer is yes, so next comment.